page_take-action.php

```diff
@@ -0,0 +1,8 @@
+<?php /* Template Name: Take Action */
+
+$page = new P4_Page();
```

Comment:
Here is where you 'd create the child class:

`$page = new P4_Take_Action_Page();

class P4_Take_Action_Page extends P4_Page {
    public __construct() {
        parent::__construct();
        $page->context['background_image']  = $page->page_meta_data['p4_background_image'][0];
    }
}`